samples/sensor/bme68x/boards/thingy53_nrf5340_cpuapp.conf

```diff
@@ -0,0 +1,11 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Segger RTT
+CONFIG_USE_SEGGER_RTT=y
```

Comment:
Why is RTT default for thingy:53 instead of UART?

Reply:
see above